NEWS.md

```
@@ -57,6 +57,11 @@ milestone for 3.8.0
     * New name and order of optional parameters.
   * Deprecated signature pgr_contraction(text,bigint[],integer,bigint[],boolean)
 
+**C/C++ code enhancement**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)